polywatch.go

```diff
@@ -42,6 +44,15 @@ func Start() error {
 		}()
 	}
 
+	cchan := make(chan os.Signal)
```

Comment:
please move signal handler to another method `func bindSignals(fn func(), ss ...os.Signal)` and pass `stop` & target signals (syscall.SIGTERM, os.Interrupt) to it

Reply:
Also it's recommended to use a buffered channel (with length of 1) in order to decrease risk of missing the signal, as it is said in documentation of `signal.Notify`

Reply:
And please rename the signals channel variable to `ntfy`